### PR TITLE
Feat(UI)/SRT-75: Keep general UI review and code cleanup

### PR DESCRIPTION
**Pull request by astanovferuz**
Stake page: Tech guide title and body font adjusted for smaller screen sizes
Stake page: "Why stake" section cards size and responsiveness adjusted for medium screens
Stake page: "Recommended staking providers" section cards title and responsiveness adjusted for medium screen sizes
Info page: "The keep solution" section cards size and responsiveness adjusted for medium screen sizes
Info page: "Securing keep" section cards size and responsiveness adjusted for medium screen sizes

### CODE REVIEW

File: src/css/info.scss [hunk 1]
```diff
@@ -91,6 +91,15 @@
     &-section {
       margin-top: 53px;
 
+      @media (max-width: #{$bp-large}) {
```
Comment (FrozenIce0617) [inline src/css/info.scss:94]: @astanovferuz , this changes should applied between medium and large size. So, the media should be:
```
@media (min-width: #{$bp-medium}) and (max-width: #{$bp-large - 1}) {
```

File: src/css/info.scss [hunk 2]
```diff
@@ -209,6 +218,20 @@
       @media (min-width: #{$bp-medium}) {
         margin-top: 86px;
       }
+
+      @media (max-width: #{$bp-large}) {
```
Comment (FrozenIce0617) [inline src/css/info.scss:222]: Same here:
```
@media (min-width: #{$bp-medium}) and (max-width: #{$bp-large - 1}) {
```

File: src/css/stake-page.scss [hunk 3]
```diff
@@ -40,6 +40,17 @@
                 min-height: 356px;
             }
         }
+        @media (max-width: #{$bp-large}) {
```
Comment (FrozenIce0617) [inline src/css/stake-page.scss:43]: Same here:
```
@media (min-width: #{$bp-medium}) and (max-width: #{$bp-large - 1}) {
```

File: src/css/stake-page.scss [hunk 4]
```diff
@@ -53,6 +64,17 @@
                 margin: 16px 0 70px;
             }
         }
+        .card   {
+            @media (max-width: #{$bp-extra-large}) {
```
Comment (FrozenIce0617) [inline src/css/stake-page.scss:68]: Same here:
```
@media (min-width: #{$bp-large}) and (max-width: #{$bp-extra-large - 1}) {
```

File: src/css/stake-page.scss [hunk 5]
```diff
@@ -97,6 +119,25 @@
                 width: 100%;
             }
         }
+        @media (max-width: #{$bp-large}) {
```
Comment (FrozenIce0617) [inline src/css/stake-page.scss:122]: Same here:
```
@media (min-width: #{$bp-medium}) and (max-width: #{$bp-large - 1}) {
```

**Review by FrozenIce0617 — APPROVED** [review]

**Review by micahjlucas — CHANGES_REQUESTED** [review]
Pausing until Robert's suggestions are confirmed.

**Review by FrozenIce0617 — APPROVED** [review]
LGTM!

**Review by micahjlucas — APPROVED** [review]

**Pull request merged**